src/fragments/lib/geo/android/maps/30_styles.mdx

```diff
@@ -39,23 +39,19 @@ You can set a different style to your map using `setStyle` method from the adapt
 <Block name="Java">
 
 ```java
-mapView.getMapAsync(map -> {
-    MapStyle mapStyle = ...; // map style from getAvailableMaps()
-    AmplifyMapLibreAdapter.setStyle(map, mapStyle, style ->
-        Log.i("MyAmplifyApp", "Finished loading " + mapStyle.getStyle())
-    );
+// where mapStyle is a reference to the selected style from Amplify.Geo.getAvailableMaps
+mapView.setStyle(mapStyle, style -> {
+    Log.i("MyAmplifyApp", "Finished loading " + mapStyle.getStyle())
 });
 ```
 
 </Block>
 <Block name="Kotlin">
 
 ```kotlin
-mapView?.getMapAsync { map ->
-    val mapStyle: MapStyle = ... // map style from getAvailableMaps()
-    AmplifyMapLibreAdapter.setStyle(map, mapStyle) {
-        Log.i("MyAmplifyApp", "Finished loading ${mapStyle.style}.")
-    }
+// where mapStyle is a reference to the selected style from Amplify.Geo.getAvailableMaps
+mapView.setStyle(mapStyle) { style ->
+    Log.i("MyAmplifyApp", "Finished loading ${mapStyle.style}.")
```

Comment:
Annotation managers like `SymbolManager` requires a reference to `MapboxMap` to initialize. The original flow of nesting `setStyle` inside of `getMapAsync` helped with that. How are we going to handle this usecase now that `setStyle` is called directly on the mapview? Also, are we just going to keep appending new features and APIs to `MapLibreView` instance? Will that be scalable?